src/content/concepts/principles-of-ax.md

```diff
@@ -0,0 +1,106 @@
+
+Digital services should host context files that users can directly provide their agents when delegating work. Examples of these hosted files: [Convex](https://docs.convex.dev/ai/using-cursor), [Supabase](https://supabase.com/docs/guides/getting-started/ai-prompts), [Netlify](https://docs.netlify.com/welcome/build-with-ai/). By doing this, customers can support their end users directly without waiting for industry standards or agentic support.
+
+The contributors to AX and these available solutions are working on improving existing approaches and pioneering new methods to solve context delivery, validation, and more.
+
+### Agent Interactivity Patterns
+
+Digital services should incorporate standardized interactivity patterns that inform agents as to how the service expects interactions with end customers to be performed. For example, there will be interactivity patterns for expressing voice/tone, design elements, and aspects of branding for agents to produce generative experiences that align with the digital service. Other patterns will set requirements for citations, content provenance, etc.
+
+Where the majority of standardized interactivity patterns will surface is for flows that are typically very sensitive in nature. Examples being: initiating purchases, authenticating users, modifying credentials, destructive actions within a digital service, etc. These patterns allow users and digital service providers to draw the boundary for which they expect agents to operate in a way that instills verifiable trust.
+
+This area is still *very* new but sites will leverage a combination of new protocols, existing protocols/patterns (e.g. oAuth and RBAC), and standardized context that can be provided to agents. **Interactivity patterns are where digital services will have hooks into maintaining continuity with end customers despite only working with their delegated agents.**
+
+### Differentiate Agent Interaction
+
+While agents should be allowed to operate on behalf of the user that delegated the work to them, their interactions should be tracked as a different medium than one operated directly by a human. At the technical layer this means metrics, logs, traces, etc. should capture that these were performed by agents. On the business layer, audit logs, service analytics, etc. should incorporate the agent identifying information.
+
+This will be critical to understanding how to support endusers with improvements to the services and the service’s AX. **Without differentiating agents in data, digital services will be harder to debug, lack context for usage audits, and will miss key insights on how agents use their service on behalf of users.**
```

Comment:
```suggestion
This will be critical to understanding how to support end users with improvements to the services and the service’s AX. **Without differentiating agents in data, digital services will be harder to debug, lack context for usage audits, and will miss key insights on how agents use their service on behalf of users.**
```